### PR TITLE
Site has been updated to account for docs move to flare (P2)

### PR DESCRIPTION
**Pull request by sigalsax**
#### What does this PR do (include background context, if relevant)?
This PR updates the secretless site by removing docs and associated links and updates them to point to the docs site on flare.
#### What ticket does this PR close?
Connected to #554 
#### Where should the reviewer start?
Navbar and footer
#### What is the status of the manual tests?
Check to ensure that there is no way to get to previous, in-house docs page from the site. Also check to ensure that there are no traces of docs documentation in repo.

#### Screenshots (if appropriate)
Before:
screen shot 2018-12-21 at 14 11 13
screen shot 2018-12-21 at 14 11 15
After:
screen shot 2018-12-21 at 14 14 59
screen shot 2019-01-31 at 13 46 07

### CODE REVIEW

File: docs/_data/navigation.yml [hunk 1]
```diff
@@ -1,109 +1,38 @@
...
-      - page: How to Deploy
-        url: /docs/get_started/how-to-deploy/overview.html
-        entries:
-        - page: Using Conjur
-          url: /docs/get_started/how-to-deploy/using-conjur.html
-        - page: Using Kubernetes Secrets
-          url: /docs/get_started/how-to-deploy/using-kubernetes-secrets.html
+footer-docs:
+  - title: Why Secretless?
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
```
Comment (sigalsax) [inline docs/_data/navigation.yml]: will remove staging from here when docs site is ready

Comment (izgeri) [reply docs/_data/navigation.yml]: @sigalsax I think it's ok to remove it now - afaik I'm the only one who'd try to click on any links from this PR, and I can manually add staging if needed to access a valid URL

File: docs/_includes/footer.html [hunk 2]
```diff
@@ -6,35 +6,35 @@
...
-								<ul class="link-col">
-										<p class="footer-title">Contribute</p>
-									<li class="footer-resource"><i class="fab fa-github"></i><a target="_blank" href="https://github.com/cyberark/secretless-broker" target="_blank">GitHub</a></li>
-									<li class="footer-resource"><i class="fas fa-comment-alt"></i><a target="_blank" href="https://groups.google.com/forum/#!forum/secretless" target="_blank">Google Group</a></li>
-								</ul>
-							</div>
+					<div class="col-md-4 col-xs-12">
+						<ul class="link-col">
+							<p class="footer-title"><a href="https://docs-staging.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm">Docs</a></p>
+							{% assign footer = site.data.navigation.footer-docs %}
```
Comment (izgeri) [inline docs/_includes/footer.html]: Are you aware that `footer-docs` is empty?

Comment (izgeri) [reply docs/_includes/footer.html]: I would either remove this section or add content

File: docs/_data/navigation.yml [hunk 3]
```diff
@@ -1,109 +1,38 @@
 main:
   - title: Docs
-    url: /docs/overview.html
+    url: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```
Comment (izgeri) [inline docs/_data/navigation.yml]: why not just https://docs.secretless.io here?

File: docs/_data/navigation.yml [hunk 4]
```diff
@@ -1,109 +1,38 @@
 main:
   - title: Docs
```
Comment (izgeri) [inline docs/_data/navigation.yml]: wdyt about changing this to `Documentation`?

File: docs/_data/navigation.yml [hunk 5]
```diff
@@ -1,109 +1,38 @@
...
+  - title: Why Secretless?
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: How Does It Work?
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/how_it_works.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Key Terms
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/key_terms.htm
+    target: target="_blank" rel="noopener noreferrer"
   - title: Reference
```
Comment (izgeri) [inline docs/_data/navigation.yml]: To be consistent with the docs site, let's change this to `Fundamentals`. I am also working on getting an updated link for this

File: docs/docs/get_started.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Get Started
 id: get_started
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/get_started.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Overview/overview.htm
```
Comment (izgeri) [inline docs/docs/get_started.md]: let's change this to link to the quick start: https://docs.secretless.io/Latest/en/Content/Get%20Started/quick_start.htm

File: docs/docs/get_started/configuration.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Configuration
 id: configuration
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/get_started/configuration.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```
Comment (izgeri) [inline docs/docs/get_started/configuration.md]: I think this page got dropped - I'm looking for a better link

File: docs/docs/get_started/configuration.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 The Secretless Broker relies on its configuration to determine which Target Services
```
Comment (izgeri) [inline docs/docs/get_started/configuration.md]: Content should be deleted

File: docs/docs/get_started/how-to-deploy/overview.md [hunk 9]
```diff
@@ -1,9 +1,9 @@
 ---
 title: How to Deploy
 id: how_to_deploy
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/get_started/how-to-deploy/overview.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```
Comment (izgeri) [inline docs/docs/get_started/how-to-deploy/overview.md]: Let's change this to https://docs.secretless.io/Latest/en/Content/Get%20Started/using-conjur.htm

File: docs/docs/get_started/how-to-deploy/overview.md [hunk 10]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 ## Overview
```
Comment (izgeri) [inline docs/docs/get_started/how-to-deploy/overview.md]: Delete content

File: docs/docs/get_started/kubernetes_tutorial.md [hunk 11]
```diff
@@ -1,867 +1,7 @@
 ---
 title: Kubernetes Tutorial
 id: kubernetes_tutorial
-layout: docs
 description: Secretless Broker Documentation
 permalink: /docs/get_started/kubernetes_tutorial.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Get%20Started/kube_tut_get_started.htm
```
Comment (izgeri) [inline docs/docs/get_started/kubernetes_tutorial.md]: this link should now be https://docs.secretless.io/Latest/en/Content/Get%20Started/kubernetes_tutorial.htm

File: docs/docs/overview.md [hunk 12]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Overview
 id: overview
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/overview.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```
Comment (izgeri) [inline docs/docs/overview.md]: should be updated to https://docs.secretless.io/Latest/en/Content/Overview/overview.htm

File: docs/docs/overview.md [hunk 13]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 The Secretless Broker is a connection broker that relieves client applications of
```
Comment (izgeri) [inline docs/docs/overview.md]: Delete content

File: docs/docs/overview/why_secretless.md [hunk 14]
```diff
@@ -1,52 +1,7 @@
 ---
 title: Why Secretless?
 id: docs
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/overview/why_secretless.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
```
Comment (izgeri) [inline docs/docs/overview/why_secretless.md]: this content was moved here: https://docs.secretless.io/Latest/en/Content/Overview/overview.htm

File: docs/docs/reference.md [hunk 15]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Reference
 id: reference
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/reference.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```
Comment (izgeri) [inline docs/docs/reference.md]: I hope to have an updated link for this that points to Fundamentals > Overview

File: docs/docs/reference.md [hunk 16]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 In the pages that follow you will find information on:
```
Comment (izgeri) [inline docs/docs/reference.md]: Delete content

File: docs/_data/navigation.yml [hunk 17]
```diff
@@ -1,109 +1,38 @@
...
-        entries:
-        - page: Using Conjur
-          url: /docs/get_started/how-to-deploy/using-conjur.html
-        - page: Using Kubernetes Secrets
-          url: /docs/get_started/how-to-deploy/using-kubernetes-secrets.html
+footer-docs:
+  - title: Why Secretless?
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: How Does It Work?
```
Comment (izgeri) [inline docs/_data/navigation.yml]: Let's remove this link.

File: docs/_data/navigation.yml [hunk 18]
```diff
@@ -1,109 +1,38 @@
...
-          url: /docs/reference/config-managers/configfile.html
-        - page: Kubernetes CRD
-          url: /docs/reference/config-managers/k8s/crd.html
-      - page: Plugin Reference
-        url: /generated/pkg_secretless_plugin_v1.html
-      - page: Health Check
-        url: /docs/reference/health-check/overview.html
+    url: https://docs-staging.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
+    target: target="_blank" rel="noopener noreferrer"
+
```
Comment (izgeri) [inline docs/_data/navigation.yml]: Please add another link to the Docs footer list that reads "Plugin Reference" and points to https://secretless.io/generated/pkg_secretless_plugin_v1.html

File: docs/_data/navigation.yml (hunk 17 above)
Comment (sigalsax) [reply docs/_data/navigation.yml]: the link For 'How Does It Work?'

File: docs/_data/navigation.yml (hunk 18 above)
Comment (sigalsax) [reply docs/_data/navigation.yml]: we haven't moved this over to the secretless docs site yet? If not, I will need to remove the sidebar (see below)
screen shot 2019-01-31 at 11 43 33

File: docs/docs/reference.md (hunk 15 above)
Comment (sigalsax) [reply docs/docs/reference.md]: :)

File: docs/docs/get_started.md [hunk 19]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 To get started trying out the Secretless Broker, you can run through our [quick start](/docs/get_started/quick_start.html)
```
[the remark this answers is not in the document]
Comment (sigalsax) [reply docs/docs/get_started.md]: good catch!

File: docs/_data/navigation.yml (hunk 18 above)
Comment (sigalsax) [reply docs/_data/navigation.yml]: pulling from plugin/v1/doc.go

Comment (izgeri) [reply docs/_data/navigation.yml]: This isn't going to move to the docs site - it's auto-generated with each build.

Comment (sigalsax) [reply docs/_data/navigation.yml]: so just leave the files in the codebase?

Comment (izgeri) [reply docs/_data/navigation.yml]: the files will continue to get auto generated on build. we just need to link to them somewhere. please add a link to the Docs footer list that reads "Plugin Reference" and points to https://secretless.io/generated/pkg_secretless_plugin_v1.html

File: docs/docs/get_started/configuration.md (hunk 7 above)
Comment (sigalsax) [reply docs/docs/get_started/configuration.md]: :)

File: docs/_data/navigation.yml (hunk 18 above)
Comment (sigalsax) [reply docs/_data/navigation.yml]: This link has already been updated under `docs/_data/navigation.yml`.

File: docs/_data/navigation.yml [hunk 20]
```diff
@@ -1,109 +1,38 @@
...
-        url: /docs/reference/health-check/overview.html
+footer-docs:
+  - title: Why Secretless?
+    url: https://docs.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Key Terms
+    url: https://docs.secretless.io/Latest/en/Content/Overview/key_terms.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Fundamentals
+    url: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```
Comment (izgeri) [inline docs/_data/navigation.yml]: https://docs.secretless.io/Latest/en/Content/Overview/how_it_works.htm

File: docs/_data/navigation.yml [hunk 21]
```diff
@@ -1,109 +1,38 @@
...
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Plugin Reference
+    url: https://secretless.io/generated/pkg_secretless_plugin_v1.html
+    target: target="_blank" rel="noopener noreferrer"
+
+footer-res:
```
Comment (izgeri) [inline docs/_data/navigation.yml]: in the interest of future clarity, it's a good idea to skip the abbreviation here and just call it `footer-resources`

File: docs/_includes/footer.html [hunk 22]
```diff
@@ -6,35 +6,35 @@
...
+							{% for link in footer %}
+									<li class="footer-resource {% if page.url == sublink.url %}active{% endif %}">
+										<a {{ link.target }} href="{{ site.baseurl }}{{ link.url }}">{{ link.title }}</a>
+									</li>
+							{% endfor %}
+					</div>
+					<div class="col-md-4 col-xs-12">
+						<ul class="link-col">
+							<p class="footer-title">Resources</p>
+							{% assign footer = site.data.navigation.footer-res %}
```
Comment (izgeri) [inline docs/_includes/footer.html]: update to `footer-resources` to match update above

File: docs/docs/get_started/configuration.md (hunk 7 above)
Comment (izgeri) [reply docs/docs/get_started/configuration.md]: https://docs.secretless.io/Latest/en/Content/docs/get_started/configuration.htm

File: docs/docs/reference.md (hunk 15 above)
Comment (izgeri) [reply docs/docs/reference.md]: https://docs.secretless.io/Latest/en/Content/Overview/how_it_works.htm

File: docs/docs/reference/providers/overview.md [hunk 23]
```diff
@@ -1,27 +1,7 @@
 ---
 title: Credential Providers
 id: providers
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/reference/providers/overview.html
```
Comment (izgeri) [inline docs/docs/reference/providers/overview.md]: Needs a redirect added to https://docs.secretless.io/Latest/en/Content/References/providers/overview.htm

File: docs/_includes/footer.html [hunk 24]
```diff
@@ -6,35 +6,35 @@
...
-							<div class="col-md-4 col-xs-12">
-								<ul class="link-col">
-										<p class="footer-title">Contribute</p>
-									<li class="footer-resource"><i class="fab fa-github"></i><a target="_blank" href="https://github.com/cyberark/secretless-broker" target="_blank">GitHub</a></li>
-									<li class="footer-resource"><i class="fas fa-comment-alt"></i><a target="_blank" href="https://groups.google.com/forum/#!forum/secretless" target="_blank">Google Group</a></li>
-								</ul>
-							</div>
+					<div class="col-md-4 col-xs-12">
+						<ul class="link-col">
+							<p class="footer-title"><a href="https://docs-staging.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm">Docs</a></p>
```
Comment (izgeri) [inline docs/_includes/footer.html]: still has a `docs-staging` link

File: docs/docs/get_started/how-to-deploy/using-conjur.md [hunk 25]
```diff
@@ -1,158 +1,7 @@
 ---
 title: How to Deploy
 id: how_to_deploy
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/get_started/how-to-deploy/using-conjur.html
+redirect_to: https://docs-staging.secretless.io/Latest/en/Content/Get%20Started/using-conjur.htm
```
Comment (izgeri) [inline docs/docs/get_started/how-to-deploy/using-conjur.md]: still has a `docs-staging` link

File: docs/docs/reference/providers/overview.md (hunk 23 above)
Comment (sigalsax) [reply docs/docs/reference/providers/overview.md]: there is one (below the comment). Is this what you meant?

File: docs/_includes/footer.html (hunk 24 above)
Comment (sigalsax) [reply docs/_includes/footer.html]: good catch!

File: docs/docs/get_started/how-to-deploy/using-conjur.md (hunk 25 above)
Comment (sigalsax) [reply docs/docs/get_started/how-to-deploy/using-conjur.md]: good catch! (again)

File: docs/docs/reference/health-check/overview.md [hunk 26]
```diff
@@ -4,6 +4,7 @@ id: healthcheck
 layout: docs
```
Comment (izgeri) [inline docs/docs/reference/health-check/overview.md]: consistent with the other pages, should this `layout` be removed?

File: docs/docs/reference/health-check/overview.md [hunk 27]
```diff
@@ -4,6 +4,7 @@ id: healthcheck
...
 description: Secretless Broker Documentation
 permalink: docs/reference/health-check/overview.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/References/health_check.htm
 ---
 
 Secretless broker exposes two endpoints that can be used for readiness and liveliness checks:
```
Comment (izgeri) [inline docs/docs/reference/health-check/overview.md]: please also remove content

File: docs/docs/reference/providers/overview.md (hunk 23 above)
Comment (izgeri) [reply docs/docs/reference/providers/overview.md]: oops. I was fooled by the way gh displayed this one. thanks

**Comment by sigalsax** [comment]
Changes pushed! What do we need the pkg_secretless_plugin_v1.html file for under  `docs/generated/pkg_secretless_plugin_v1.html` ? It seems to be the only on calling for the docs layout (I have since changed that 'subpages') but it looks empty and funky. Is this what we want? See below:

Pkg Plugin:
screen shot 2019-02-28 at 13 52 55

Should this be deleted as well?

**Comment by izgeri** [comment]
@sigalsax that page only gets generated when you run `./bin/build_website` - try running that and then viewing this branch with `bundle exec jekyll serve`

File: docs/docs/get_started/kubernetes_tutorial.md [hunk 28]
```diff
@@ -226,7 +226,6 @@ spec:
       nodePort: 30001
   type: NodePort
 EOF
-
```
Comment (izgeri) [inline docs/docs/get_started/kubernetes_tutorial.md]: can we restore the newlines that were deleted? I think they were all in there to improve readability of commands in code blocks

**Pull request merged**